src/common/Section/styled.js

```diff
@@ -9,18 +9,22 @@ export const StyledSection = styled.div`
    
     @media (max-width: ${bpMobile}px) {
         padding: 24px 16px 0 16px;
+        max-width: 640px;
   }
 `;
 
 export const Title = styled.h1`
     margin: 0 0 24px 0;
+    width: 100vh;
```

Comment:
I think you should remove this line. It causes trouble on narrow screens:
![obraz](https://user-images.githubusercontent.com/118211320/220995427-773e853a-e475-4018-9131-52043c947bf9.png)